packages/inter-protocol/docs/governance.md

```diff
@@ -70,3 +96,18 @@ In `packages/vats/src/provisionPool.js`:
 | Governance Key          | Type    | WP? |
 | ----------------------- | :------ | --- |
 | PerAccountInitialAmount | Amount  | N/A |
+
+### Auctioneer
+
+([source](
+https://github.com/Agoric/agoric-sdk/blob/5cbc847618dfb22f80e4641e5221b80e6daa109a/packages/inter-protocol/src/auction/params.js#L58-L68)))
+
+| Governance Key            | Type          | WP? |
+| ------------------------- | :------------ | --- |
+| StartFrequency            | RelativeTime  | N/A |
+| ClockStep                 | RelativeTime  | N/A |
+| StartingRate              | BasisPoints   | N/A |
+| LowestRate                | BasisPoints   | N/A |
+| DiscountStep              | BasisPoints   | N/A |
```

Comment:
If we're laying an interpretation NatValue here, should we do likewise for ChargingPeriod, RecordingPeriod above? Those are relative times in seconds, yes?